packages/application-shell/src/components/navbar/navbar.mod.css

```diff
@@ -70,15 +73,24 @@
   width: var(--icon-size);
   height: var(--icon-size);
 }
+.list-item .icon > svg *:not([fill='none']) {
+  /* stylelint-disable-next-line csstools/value-no-unknown-custom-properties */
+  fill: var(--color-for-navbar-icon);
+}
+.list-item .icon__active > svg *:not([fill='none']) {
+  /* stylelint-disable-next-line csstools/value-no-unknown-custom-properties */
+  fill: var(--color-for-navbar-icon-when-active);
+}
```

Comment:
We're overwriting the fill colors directly here (similarly to what the icon itself does internally) to be able to use the design tokens, even though the icon component has a color prop.